sounddevice.py

```diff
@@ -587,6 +587,38 @@ def get_status():
         raise RuntimeError('play()/rec()/playrec() was not called yet')
 
 
+def get_stream():
+    """Get a reference to the current Stream.
```

Comment:
What is the reason to capitalize "Stream" (also in the next line)?

I'm just asking because I'm not a native speaker ...